src/files-card-body.jsx

```diff
@@ -191,7 +191,7 @@ export const FilesCardBody = ({
             if (ev.detail > 1) {
                 onDoubleClickNavigate(file);
             } else {
-                if (!ev.ctrlKey || selected === path[path.length - 1]) {
+                if (!ev.ctrlKey) {
```

Comment:
There's no proptypes here -- is that a boolean then?

Reply:
I.e. is selected a flag or a string (with a path) -- both sounds plausible just by name. That's why `proptypes` are useful,  plus of course comments what the properties actually do.

Reply:
`selected` is a list of objects of FolderFileInfo. So comparing an list to a path here makes no sense at all. That is what Typescript points out when you type it.

FYI: we discussed making `selected` a string of the `file` and then look it up via `files` instead.